src/js/site/main.js

```diff
@@ -395,3 +395,17 @@ $(document).ready(function () {
       }
     });
 });
+
+// insights page column highlights
+// highlight will only work if inside of #inights-table
+$(document).ready(function () {
+  var urlParts = window.location.href.split('#');
+  if (urlParts && urlParts[1]) {
+    var highlight = urlParts[1].split('?');
```

Comment:
lets us either `let` or `const` since we have es6 now 😄

Reply:
Good catch!